### ISSUE TITLE
Privacy Notice from payments page is incorrectly highlighted after clicked

### ISSUE THREAD

**Issue by bsurd** [mozilla/fxa#3349]
#### Affected versions:
- All Firefox

#### Affected Platforms:
- Windows 10 (64-bit)

#### Steps to reproduce
1. Go to the [Full-Device VPN](https://stage.guardian.nonprod.cloudops.mozgcp.net/vpn) page.
2. Click on the "Subscribe Now" button and make an account.
3. Click on the "Privacy Notice" or select it via the keyboard.

#### Expected result:
- The "Privacy Notice" is properly highlighted.

#### Actual result:
- The "Privacy Notice" is not properly highlighted.

![image](https://user-images.githubusercontent.com/17178945/68749934-b89c2500-0607-11ea-84a7-41322a1d7a4b.png)

**Comment by clouserw** [comment]
@juliastarkov can you clarify what this should look like?  It sounds like we want the border to look like that, but it needs some padding to keep the 'e' from running into the edge?

**Comment by juliastarkov** [comment]
@clouserw I did not spec out the border to look that way; do you know where that design was specified?

Currently on FxA, hovering just shows an underline (as seen in image below) and clicking makes the weird border pop up, but only sometimes. I think we should remove this border altogether (clicking or hovering). Hover should be underlined. 

<img width="303" alt="Screen Shot 2019-11-18 at 8 55 31 AM" src="https://user-images.githubusercontent.com/26334275/69073105-9a586f80-09e1-11ea-9cb9-7caecc7ef738.png">

**Comment by clouserw** [comment]
We were in a triage meeting so I'm not sure.  If underline is the right thing, that works for me.  Thanks

**Comment by juliastarkov** [comment]
@clouserw Yes, we can set the underline and the right hand side padding exactly how it is currently in FxA. Let me know if you need more specific specs.

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

@juliastarkov can you clarify what this should look like?  It sounds like we want the border to look like that, but it needs some padding to keep the 'e' from running into the edge?

**Comment by juliastarkov** [comment]
@clouserw I was pinged about this issues - is this still unresolved?

**Comment by clouserw** [comment]
Hi @juliastarkov .  That shouldn't have pinged you and I don't know why it did.  We're experimenting with a syncing robot that apparently malfunctioned.  Sorry for the bother!
